test/images/gen_sifs.go

```diff
@@ -170,6 +181,12 @@ func generateImages() error {
 				objectSBOM,
 			},
 		},
+		{
+			path: "one-object-oci-blob-config.sif",
```

Comment:
Since we likely only need one "OCI blob" SIF for testing, lets keep the name generic.

```suggestion
			path: "one-object-oci-blob.sif",
```